frontend-public/src/utils/helpers.js

```diff
@@ -44,3 +51,15 @@ export const createLabelHash = (obj) =>
 
 // Function to format an API date string to human readable
 export const formatDate = (dateString) => moment(dateString, "YYYY-MM-DD").format("MMM DD YYYY");
+
+export const downloadFileFromDocumentManager = (docManagerGuid, filename) => {
+  if (!docManagerGuid || !filename) {
+    throw new Error("Must provide both docManagerGuid and filename");
+  }
+
+  // TODO: Update url when Document Manager moves to its own microservice.
+  const url = `${ENVIRONMENT.apiUrl + DOCUMENT_MANAGER_FILE_GET_URL}/${docManagerGuid}`;
+  axios.get(url, { responseType: "arraybuffer", ...createRequestHeader() }).then((response) => {
```

Comment:
this seems to be more of an actionCreator than a util function.

Reply:
I wasn't sure if it should be an action creator or not. This isn't touching the store or modifying any state, which is the normal definition of an action creator.

Reply:
After discussion: Move the method into a new utils file, as its neither a static helper function nor an action creator.